src/style.css

```diff
@@ -1,3 +1,74 @@
+html {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: center;
+  height: 60px;
+  margin: 0 10px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  line-height: 60px;
+  margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the 'Roboto' font is correctly loaded in the HTML file. This is important for the font-family to render as expected.